package.json

```
@@ -8,7 +8,7 @@
     "scripts": {
         "start": "npm run build && cd examples && npm run serve-example",
         "install-examples": "npm install && cd examples && npm install",
-        "gh-build": "npm-run-all processCSS build-example",
+        "gh-build": "npm-run-all processCSS && cd examples && npm run build-example",
```

nit: you don't need the rpm-run-all here since the other scripts are using the `&&`
```suggestion
        "gh-build": "npm run processCSS && cd examples && npm run build-example",
```